utils/rpcd-mod-packagelist/Makefile

```diff
@@ -0,0 +1,40 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rpcd-mod-packagelist
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+PKG_LICENSE:=GPL-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/rpcd-mod-packagelist
+  SECTION:=utils
+  CATEGORY:=Base system
+  TITLE:=ubus packagelist
+  MAINTAINER:=Paul Spooren <paul@spooren.de>
+  DEPENDS:=rpcd @!CLEAN_IPKG
+endef
+
+define Package/rpcd-mod-packagelist/description
+	add ubus call to receive user installed packages without the need of opkg installed
```

Comment:
In general, (and in no way endorsing that this should be ramme dinto 17.01) given that this package has _no_ other documentation, It would probably be best to add the usage guide here.  "add ubus call" isn't very descriptive.

If this is _actually_ only meant to be used by some other package, just include it there.  If it's _meant_ to be used/available for other people, then you should provide documentation showing how it is intended to be used, somewhere!